WordPress/Classes/ViewRelated/Notifications/Views/NotificationSettingsSiteView.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import SwiftUI
+
+struct NotificationSettingsSiteView: View {
+    let viewModel: NotificationSettingsSiteViewModel
+
+    var body: some View {
+        HStack(alignment: .center, spacing: 16) {
+            SiteIconView(viewModel: viewModel.icon)
+                .frame(width: 32, height: 32)
+
+            VStack(alignment: .leading) {
+                Text(viewModel.title)
+                    .font(.body)
+                    .lineLimit(1)
+
+                Text(viewModel.details)
+                    .font(.footnote)
+                    .foregroundStyle(.secondary)
+                    .lineLimit(1)
+            }
+        }
+    }
+}
+
+struct NotificationSettingsSiteViewModel {
+    let title: String
+    let details: String
+    let icon: SiteIconViewModel
+
+    init(blog: Blog) {
+        self.title = blog.title ?? "–"
```

Comment:
There is a slight code difference here comparing to the original code. Just want to double check if this change is intentional.

```
cell.textLabel?.text = settings.blog?.settings?.name ?? settings.channel.description()
```

Reply:
I missed it, but `settings.channel.description()` simply [returns](https://github.com/wordpress-mobile/WordPress-iOS/blob/7dfa0f8a193477904962dc0a282a6089a94fdaef/WordPress/Classes/Models/Notifications/NotificationSettings.swift#L93) `NSLocalizedString("WordPress Blog", comment: "Settings for a Wordpress Blog")`, which I'm not sure is a good fallback value. If the is no title, I'd rather it show "–".

Reply:
There is another difference: `blog.title` and `blog.setting.name`, but I'm not sure if their values are actually different or not.